pjlib/include/pj/string.h

```diff
@@ -69,6 +69,12 @@ PJ_BEGIN_DECL
  *  - @ref page_pjlib_string_test
  */
 
+/* Check if a string is truncated and if yes, put a suffix of ".."
+ * to indicate the truncation.
+ */
```

Comment:
also mention that the macro is for checking truncated string in pj_ansi_snprintf()?